string name
string[] type
uint32 size
uint32[] indices
float32[] weight
float32 lambda
float32 lambda2
string activation_state
string[] disabled_joints
string lib_name
